moonstone/core/plot_base.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import pandas as pd
+import pylab as plt
+from matplotlib import pyplot
+
+
+class PlotTemplateDataFrame:
```

Comment:
Maybe it would be worth creating a new module within moonstone called `plot` following the same structures as other modules and think about a common way the `plot` module should work